packages/react/src/components/DataTable/TableExpandHeader.js

```diff
@@ -57,18 +72,18 @@ TableExpandHeader.propTypes = {
    * Specify the string read by a voice reader when the expand trigger is
    * focused
    */
-  ariaLabel: PropTypes.string.isRequired,
+  ariaLabel: isRequiredIfPropExists('ariaLabel'),
 
   /**
    * Specify whether this row is expanded or not. This helps coordinate data
    * attributes so that `TableExpandRow` and `TableExapndedRow` work together
    */
-  isExpanded: PropTypes.bool.isRequired,
+  isExpanded: isRequiredIfPropExists('isExpanded'),
```

Comment:
Would this be like:

```suggestion
  isExpanded: isRequiredIfPropExists('enableExpando'),
```

?

Reply:
Oh right, let me fix